src/Sandbox/FPSCounter.cpp

```diff
@@ -0,0 +1,58 @@
+#include "FPSCounter.h"
+
+#include <string>
+#include <GLFW/glfw3.h>
+
+#include "Utils/EngineExceptions.h"
+
+FPSCounter::FPSCounter()
+{
+    if (!gltInit())
+	{
+		throw BadInitializationException("Failed to initialize GLText library!");
+	}
+
+    this->_lastTime = glfwGetTime();
```

Comment:
We rather don't use "this" keyword in the project, but it's only a suggestion to remove it.